web-common/src/components/chip/core/Chip.svelte

```diff
@@ -11,7 +11,8 @@
   export let removable = false;
   export let active = false;
   export let readOnly = false;
-  export let type: "measure" | "dimension" | "time" | "special" = "dimension";
+  export let type: "measure" | "dimension" | "time" | "special" | "guest" =
```

Comment:
It's not great that our business logic is seeping into our design system. Rather than considering this chip `type: guest`, can we consider it `color: amber` (and, if needed, something like `size: XXX` and `rounded: true`)?